behat/tests/features-v2/client-management/admin/view-client.feature

```diff
@@ -0,0 +1,23 @@
+@v2 @admin-client-view
+Feature: Admin - View client details
+
+  Scenario: An admin user views client details associated with a Lay deputy
+    Given an admin user accesses the admin app
+    When I visit the client details page for an existing client linked to a Lay deputy
+    Then I should see the clients court order number
+    And I should see the Lay deputies name, address and contact details
+    And I should see the reports associated with the client
```

Comment:
I really like the function for this. I think we want to add a check for that the manage link takes us through to the /admin/report/<id>/manage page or is that a separate PR?

Reply:
Yeah I was on the fence as this is more about viewing the client...but at the same time I'm not sure where it should sit in another suite. Maybe `report-management` as we can unsubmit them?